bin/docker-build

```diff
@@ -2,20 +2,8 @@
 
 set -eu
 
-. bin/_docker.sh
-. bin/_tag.sh
-
-if [ $# -eq 0 ]; then
-    tag="$(clean_head_root_tag)"
-elif [ $# -eq 1 ]; then
-    tag="${1:-}"
-else
-    echo "usage: $(basename $0) [tag]" >&2
-    exit 64
-fi
-
-bin/docker-build-controller  "$tag"
-bin/docker-build-web         "$tag"
-bin/docker-build-proxy       "$tag"
-bin/docker-build-proxy-init  "$tag"
-bin/docker-build-cli         "$tag"
+bin/docker-build-controller
```

Comment:
can we error when `[ $# -ne 0 ]`?  This shouldn't work if you give it a tag on the commandline.